docs/transactions.md

```diff
@@ -219,3 +219,26 @@ sequelize.transaction({
 The `transaction` option goes with most other options, which are usually the first argument of a method.
 For methods that take values, like `.create`, `.update()`, `.updateAttributes()` etc. `transaction` should be passed to the option in the second argument.
 If unsure, refer to the API documentation for the method you are using to be sure of the signature.
+
+## Locks
```

Comment:
Can improve this by adding more examples from [here](https://github.com/sequelize/sequelize/blob/a7e9e2b8ab978230e3f8e39ffdce2a695a102d00/lib/transaction.js#L252-L271)